frontend/pulse3d/src/components/interactiveAnalysis/WaveformGraph.js

```diff
@@ -413,13 +413,6 @@ export default function WaveformGraph({
             peaksWindowLine.attr("x2", endPosition);
           })
           .on("end", function () {
-            const timeWidth = d3.select(this).attr("width");
```

Comment:
Just wondering about what is replacing this?

Reply:
I was working on removing extra state changes while fixing a bug and tested without these. I'm not sure why they were added but I think they are not needed now because of other IA changes. When removing the state changes IA seems to be functioning normally.

Reply:
The state for `newStartTime/newEndTime` time is updated on lines `823/828 `, and in the use effect on line `876`

Reply:
https://user-images.githubusercontent.com/84344924/236977846-2cd86c52-3b64-4b2a-be37-9047d85c2c28.mov

this functionality is being affected by the removal

Reply:
there is different functionality for dragging an individual time line and dragging the entire fill box. Lines 823/828 are for individual lines, this removed functionality for the entire fill box

Reply:
I did not realize that  the whole window could be dragged. Thank you for the explanation.